features/256-auth-gear-interaction-overview-in-next/api.md

```diff
@@ -93,7 +95,9 @@ if err != nil {
     return response
 }
 
-err = hooks.ExecuteAfterHooks(user)
+hooks.ExecuteAfterAsyncHooks(user)
+
+err = hooks.ExecuteAfterSyncHooks(user)
```

Comment:
hooks.ExecuteAfterSyncHooks should goes BEFORE Async hooks?

Because if the Sync Hooks is failed, Async hooks may not want to be triggered.